.env-template

```diff
@@ -11,3 +11,4 @@ SNOWFLAKE_WAREHOUSE=<add snowflake warehouse here>
 AWS_ACCESS_KEY_ID=<add aws api key here>
 AWS_SECRET_ACCESS_KEY=<add aws secret key here>
 AIRFLOW_VAR_FOO=templated_file_name
+ASTRO__DATAFRAME__ALLOW_UNSAFE_STORAGE=True
```

Comment:
@dimberman We should use prefix - `AIRFLOW__ASTRO_SDK__` for env variables since we have standardized the config retrieval from airflow config manager. PR - https://github.com/astronomer/astro-sdk/issues/503

Reply:
@tatiana i think the size limit idea would lead to very weird user interactions (e.g. someone is using the library and their DAGs suddenly start failing because their dataframe is too large).

Storing data in the metadata DB is an anti pattern we shouldn't promote